core/iwasm/interpreter/wasm.h

```diff
@@ -213,10 +213,11 @@ typedef struct WASMTag WASMTag;
 
 #ifndef WASM_VALUE_DEFINED
 #define WASM_VALUE_DEFINED
+
 typedef union V128 {
```

Comment:
I am not sure about that but it's a little confused to keep both `union V128` and `union wasm_v128_t`, especially when they are sharing the same definition.

Reply:
I'm happy to change them to use the same name. Which do you prefer? I'm not completely sure about the naming rules, but using the name V128 in wasm_export.h didn't feel correct.